vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -13,40 +13,37 @@
 # it can get out of sync. But probably we don't want the docstrings in the short code snippet.
 # Ultimately these charts will probably move to vizro.charts anyway.
 @capture("graph")
-def butterfly(data_frame: pd.DataFrame, x1: str, x2: str, y: str) -> go.Figure:
+def butterfly(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
     """Creates a custom butterfly chart using Plotly's go.Figure.
 
     A butterfly chart is a type of bar chart where two sets of bars are displayed back-to-back, often used to compare
     two sets of data.
 
     Args:
-        data_frame (pd.DataFrame): The data source for the chart.
-        x1 (str): The name of the column in the data frame for the first set of bars (negative values).
-        x2 (str): The name of the column in the data frame for the second set of bars (positive values).
-        y (str): The name of the column in the data frame for the y-axis (categories).
+        data_frame (pd.DataFrame): The data frame for the chart. Can be long form or wide form.
+            See https://plotly.com/python/wide-form/.
```

Comment:
Why do you emphasise long vs wide form for these four added examples?

Reply:
* it's not something that would be immediately obvious from the examples given. Because we only have one example for each chart we only show the `data_frame` that's in either long or wide form, so you might think that your data has to be in that format
* it's like a super power of plotly express that this is handled so well so something that we should boast about 😀